https://github.com/Field-Robotics-Lab/ds_sim/pull/6/commits/0604360f456a2974dd53ceb6300ed5f8e264c38b broke something. Rolling back to a commit before compiling ok.

 ```bash
Errors     << ds_sim:make /home/woensug/uuv_ws/logs/ds_sim/build.make.016.log
In file included from /home/woensug/uuv_ws/src/ds_sim/gazebo_src/dsros_dvl.cc:42:
/home/woensug/uuv_ws/src/ds_sim/gazebo_src/dsros_dvl.hh:60:10: fatal error: dave_gazebo_world_plugins/StratifiedCurrentVelocity.pb.h: No such file or directory
   60 | #include <dave_gazebo_world_plugins/StratifiedCurrentVelocity.pb.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/dsros_sensors.dir/build.make:102: CMakeFiles/dsros_sensors.dir/gazebo_src/dsros_dvl.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:3206: CMakeFiles/dsros_sensors.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
cd /home/woensug/uuv_ws/build/ds_sim; catkin build --get-env ds_sim | catkin env -si  /usr/bin/make --jobserver-auth=3,4; cd -
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,11 +53,10 @@ endforeach()
 
 catkin_package(
   INCLUDE_DIRS include
-  #LIBRARIES ds_sim_gazebo_msgs dsros_jointstate_publisher dsros_ros_depth dsros_ros_ins dsros_ros_dvl dsros_ros_gps
-  LIBRARIES dsros_sensors
+  # Export for downstream packages
+  LIBRARIES ds_sim_gazebo_msgs dsros_jointstate_publisher dsros_ros_depth dsros_ros_ins dsros_ros_dvl dsros_ros_gps
   CATKIN_DEPENDS xacro roscpp gazebo_ros sensor_msgs ds_core_msgs ds_sensor_msgs ds_actuator_msgs ds_multibeam_msgs
     dave_gazebo_world_plugins
-  DEPENDS
 )
 
 # Build any protobuf messages we're going to use

--- a/package.xml
+++ b/package.xml
@@ -9,7 +9,7 @@
   <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
   <license>BSD</license>
 
-  <url type="repository">https://bitbucket.org/whoidsl/ds_sim</url> -->
+  <url type="repository">https://bitbucket.org/whoidsl/ds_sim</url>
 
   <author email="ivaughn@whoi.edu">Ian Vaughn</author>
   <author email="ssuman@whoi.edu">Stefano Suman</author>

